### ISSUE TITLE
Rewrite README.md 

### ISSUE THREAD

**Issue by kakawait** [kakawait/sonar-gitlab-plugin#15]

**Comment by dve** [comment]
Just if someone else searches for this: its not `sonar.gitlab.commit_sha` anymore but  `sonar.gitlab.commit_hashes`

**Comment by kakawait** [comment]
Yes sorry I know that actually I incubate it inside my company. I will rewrite readme and many fixes soon. (Before end of the next week).

Just after 1.8 release I will maintain it more

**Comment by dve** [comment]
Just wrote it here to not forget it ;-) 
Great that you take care of the plugin! Do you need help at testing?

Do you have a way to set all hashes for the build in the .gitlab-ci.yml file?

**Comment by kakawait** [comment]
@dve actually i'm using that to set `hashes`

```
-Dsonar.gitlab.commit_hashes=$(git log --pretty=format:%H origin/master..$CI_BUILD_REF | tr '\n' ',')
```

**Comment by kakawait** [comment]
fixes in #19
